packages/nest/README.md

```diff
@@ -114,15 +127,19 @@ import { FeatureClient } from '@openfeature/nestjs-sdk';
 export class OpenFeatureTestService {
   constructor(
     @FeatureClient() private defaultClient: Client,
-    @FeatureClient({ name: 'differentServer' }) private namedClient: Client,
-  ) {
-  }
+    @FeatureClient({ name: 'differentProvider' }) private namedClient: Client,
+  ) {}
 
-  public async getMessage() {
-    const companyName = await this.defaultClient.getStringValue('companyName', 'Unknown Company');
-    return `Hey User from ${companyName}`;
+  public async getBoolean() {
+    return await this.defaultClient.getBooleanValue('testBooleanFlag', false);
   }
 }
 ```
 
+## Module aditional information
+
+### Flag evaluation context injection
 
+Whenever a flag evaluation occours, context can be provided with information like user e-mail, role, targeting key, etc in order to trigger specific evaluation rules or logic. The `OpenFeatureModule` provides a way to configure context for each request using the `contextFactory` option.
+The `contextFactory` is ran in a NestJS interceptor scope to configure the evaluation context and than it is used in every flag evaluation related to this request.
```

Comment:
We should use `Nest`, `Nest.js` or `NestJS` consistently.
I would go with one of the first two.

Reply:
Based on the docs, I see [NestJS](https://docs.nestjs.com/)

Reply:
I would say there is even more `Nest`, but `NestJS` seems fine.
At least the second was wrong 😅

Reply:
I would say there is even more `Nest`, but `NestJS` seems fine.
At least the second was wrong 😅

Reply:
I mostly care that we're consistent.

Reply:
Alright, 

I'll keep using NestJS since the package is `@openfeature/nestjs-sdk`